.github/workflows/update-browserslist.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Browserslist DB
+on:
+  - workflow_call
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
+        npm: [8.5.x]
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+      
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
+    
+    - name: Install dependencies
+      run: npm install -g npm@${{ matrix.npm }}
+
+    - name: Install Packages 
+      run: npm ci
+
+    - name: Update dependencies
+      run: npx browserslist@latest --update-db
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v4
```

Comment:
@sarina and @feanil, mind taking a peek at this PR (and [the motivation](https://github.com/openedx/frontend-wg/issues/101))?  This is the critical line.

What's happening here is we're substituting [a custom edx/testeng-ci PR creator job](https://github.com/openedx/frontend-app-account/pull/642/files#diff-3f8e8ba88da826420f93bbaeff9dea2c0ee9b77aa35858122d6341ef013d5ec9R34) repo with a Github workflow.

It looks to me like a good precedent for automatic PR creation.

Reply:
This seems reasonable, I'm not sure what you mean by this being the "critical line". The create-pull-request action @ v4 seems good.

Reply:
Yeah, that's what I meant: to check that action looks good. :)